### ISSUE TITLE
Area calculation failure

### ISSUE THREAD

**Issue by artemp** [mapnik/mapnik#3402]
```c++
#include <iostream>
#include <vector>

struct point
{
    point(double _x, double _y)
        : x(_x), y(_y) {}
    double x;
    double y;
};

template <typename T>
bool is_clockwise(T const& ring)
{
    double area = 0.0;
    std::size_t num_points = ring.size();
    for (std::size_t i = 0; i < num_points; ++i)
    {
        auto const& p0 = ring[i];
        auto const& p1 = ring[(i + 1) % num_points];
        area += p0.x * p1.y - p0.y * p1.x;
    }
    std::cerr << "AREA=" << area << std::endl;
    return (area < 0.0) ? true : false;
}

int main( int argc, char ** argv)
{
    std::vector<point> ring;
    ring.emplace_back(-13499697.0366658326, 4698431.85179749783);
    ring.emplace_back(-13499697.1113113686, 4698431.85179749783);
    ring.emplace_back(-13499697.0366658326, 4698431.92644303292);
    ring.emplace_back(-13499697.0366658326, 4698431.85179749783);

    std::cerr << "Is clockwise? " << is_clockwise(ring) << std::endl;

    return 0;
}
```

**Comment by artemp** [comment]
Hmm .. `boost::geometry::area` returns 0 as well for this triangle.

**Comment by artemp** [comment]
Turned out this has been fixed in 1.61 - https://svn.boost.org/trac/boost/ticket/11928

**Comment by artemp** [comment]
fixed in 66160e9
